specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-03-01/managedClusters.json

```diff
@@ -1997,7 +1997,7 @@
           "type": "object",
           "properties": {
             "balance-similar-node-groups": {
-              "type": "boolean"
+              "type": "string"
```

Comment:
This change is already in master:

https://github.com/Azure/azure-rest-api-specs/blob/master/specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-03-01/managedClusters.json#L1999

This change is already in master. I think the dev branch might need rebasing but we don't have access to that.

Reply:
cc @fengzhou-msft/anyone who has access